package.json

```diff
@@ -80,7 +80,7 @@
     "gl-plot2d": "^1.3.1",
     "gl-plot3d": "^1.5.7",
     "gl-pointcloud2d": "^1.0.1",
-    "gl-scatter3d": "^1.0.11",
+    "gl-scatter3d": "git://github.com/gl-vis/gl-scatter3d.git#2ee9fbb3c29af694ad64258a27dc04d617896e84",
```

Comment:
https://github.com/gl-vis/gl-scatter3d/compare/master...issue-704

Reply:
BTW now that you have PRs in these other repos, even better than the branch link would be to link to that PR directly
https://github.com/gl-vis/gl-scatter3d/pull/9

Reply:
Thanks for the note. But the other PR on the gl-scatter3d repo is not opened by me.

Reply:
My apologies @alexcjohnson, 
Here is the link to the open PR:
https://github.com/gl-vis/gl-scatter3d/pull/9